src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -67,14 +67,11 @@
 					</NcActionText>
 					<!-- Edited message timestamp -->
 					<NcActionButtonGroup v-if="lastEditTimestamp">
-						<NcActionText>
+						<NcActionText class="editor-timestamp">
 							<template #icon>
 								<ClockEditOutline :size="16" />
 							</template>
-							{{ lastEditActorDisplayName }}
-						</NcActionText>
-						<NcActionText>
-							{{ editedDateTime }}
+							{{ lastEditActorDisplayName + ' • ' + editedDateTime }}
 						</NcActionText>
```

Comment:
My suggestion would be:
```
					<NcActionText v-if="lastEditTimestamp"
						:name="lastEditActorDisplayName">
						<template #icon>
							<ClockEditOutline :size="16" />
						</template>
						{{ editedDateTime }}
					</NcActionText>
```

![Bildschirmfoto vom 2024-01-25 17-29-37](https://github.com/nextcloud/spreed/assets/213943/891e26e1-5318-4be9-9f79-1d0d39ca35c4)